uuid/v5.ts

```diff
@@ -53,8 +53,8 @@ export async function generate(
   const buffer = await crypto.subtle.digest("sha-1", toHash);
   const bytes = new Uint8Array(buffer);
 
-  bytes[6] = (bytes[6] & 0x0f) | 0x50;
-  bytes[8] = (bytes[8] & 0x3f) | 0x80;
+  bytes[6] = (bytes[6]! & 0x0f) | 0x50;
+  bytes[8] = (bytes[8]! & 0x3f) | 0x80;
```

Comment:
line 53 crypto.subtle.digest always returns an ArrayBuffer of length 20, so bytes 6 and 8 will be fine.